packages/experiments/src/components/CollapsibleSection/CollapsibleSection.ts

```diff
@@ -0,0 +1,33 @@
+import { createComponent, createComponentWithState } from '../../Foundation';
+import { CollapsibleSectionView } from './CollapsibleSection.view';
+import { CollapsibleSectionState } from './CollapsibleSection.state';
+import { getStyles as styles } from './CollapsibleSection.styles';
+import {
+  ICollapsibleSectionProps,
+  ICollapsibleSectionControlledProps,
+  ICollapsibleSectionViewProps,
+  ICollapsibleSectionStyles
+} from './CollapsibleSection.types';
+
+// TODO: can we use controlled as arg instead of having two distinct types? maybe controlled could be used by createComponent
+//          to specify whether or not to use the state component? we could even have default controlled types, like toggle and counter
+export const CollapsibleSectionUncontrolled: React.StatelessComponent<
```

Comment:
Separate components for Controlled vs Uncontrolled; please remove

Reply:
I was already in the process. 😆  I have added a TODO in #5721 for addressing controlled vs. uncontrolled behavior with state objects, ideally not requiring separate state components for every component.